bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,17 @@
 	<description>The OpenWebNet Binding integrates the BTicino/Legrand 'MyHOME' connected home system using the OpenWebNet
 		protocol. It supports BUS (SCS) and Zigbee USB gateways and devices.</description>
 	<connection>local</connection>
+
+	<discovery-methods>
+		<discovery-method>
+			<service-type>upnp</service-type>
+			<match-properties>
+				<match-property>
+					<name>manufacturer</name>
+					<regex>(?i).*BTICINO.*</regex>
```

Comment:
From the logs, actual strings that are found are:

`BTicino S.p.A.`
or
`BTicino S.p.A. (http://www.bticino.it)`
If the (?i) part of the regex ignores the case, the this should work